### ISSUE TITLE
Content fail to load when document is opened without passing through fileManager

### ISSUE THREAD

**Issue by amoshydra** [nus-mtp/lecture-note-2.0#313]
`fileManager.getFileByID` expect `fileID` to be an integer.
However a number string is passed into the function instead.
```javascript
getFileByID: function getFileByID(fileID) {
  return core.getFile(fileID);
},
```

### Example: 
loading https://sashimi-note-dev.herokuapp.com/content?id=1 directly will cause the error to occur.

### Resolution:
Either have the conversion done at the `FileManager.vue` or inside the `fileManager` logic.

**Comment by amoshydra** [comment]
Having `fileManager` logic to do the conversion could make the API friendlier to use. 
@tayjiehao Do you foresee any concern [parsing the string to integer](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/parseInt) at `fileManager` logic?
If not, this should be the preferred way.

**Comment by tayjiehao** [comment]
The only major concern is to receive a string that do not contain a number for it to be parsed into an integer. Other than that, `Filemanager` can handle the logic to do the conversion. An error can be thrown if the parsing of string to integer fail.

**Comment by tayjiehao** [comment]
Fixed in #335

**Comment by tayjiehao** [comment]
`fileManger.getFileByID()` and `fileManager.getFolderByID()` should accept both `Number` and `String` data types

**Comment by tayjiehao** [comment]
Fixed in #337
